pyro/distributions/hmm.py

```diff
@@ -942,6 +942,9 @@ def __init__(self, initial_dist, transition_matrix, transition_dist,
             else:
                 break
         if not observation_dist.event_shape:
+            if isinstance(observation_dist, torch.distributions.Normal) and \
+                    not isinstance(observation_dist, TorchDistributionMixin):
+                observation_dist = Normal(observation_dist.loc, observation_dist.scale)
```

Comment:
workaround observation_dist is LogNormal

Reply:
Can you add that as a comment?

We should really fix `pyro.distributions.torch.LogNormal`. Another option is to force:
```py
if type(observation_dist) is torch.distributions.Normal):
    observation_dist.__class__ = dist.Normal
```

Reply:
Sounds reasonable, I'll do that.